src/pages/not-found/not-found.tsx

```diff
@@ -0,0 +1,27 @@
+import Layout from '../../components/layout';
+import styles from './not-found.module.scss';
+import { images } from '../../utils';
+import Button from '../../components/button';
+
+const NotFoundPage = () => {
+  return (
+    <Layout title="">
```

Comment:
Hmm it's not great to provide an empty string here as it will result in the page having an empty `<h1>`. Probably makes sense to make the title an optional prop.

Reply:
Actually I see now that we may always need to have a title, so that we pass it inside Helmet in `layout.tsx`. So it's probably best I pass a flag to hide the h1 whenever not needed

```ts
      <Helmet>
        <title>{`API3 DAO | ${title}`}</title>
      </Helmet>
```